framework/Browser.js

```diff
@@ -142,6 +142,11 @@ class Browser {
     Logger.debug(`Sending text to alert: ${text}`)
     await browser.sendAlertText(text)
   }
+
+  async waitUntil(condition, { timeout = 5000, interval = 500 } = {}) {
```

Comment:
pls use constants instead
besides, you may want to have a method which won't throw an error if condition not met but return false in this case.